internal/verifier/migration_verifier.go

```diff
@@ -76,6 +76,13 @@ const (
 	notOkSymbol = "\u2757" // heavy exclamation mark symbol
 )
 
+type clusterType string
+
+const (
+	srcReaderType clusterType = "source"
```

Comment:
“cluster type” seems relatively non-descript: “type” often means topology, for example.

Maybe `whichCluster`?